.github/workflows/ci.yml

```diff
@@ -23,6 +23,9 @@ jobs:
       - name: Install uv
         uses: astral-sh/setup-uv@v5
 
+      - name: Install Pyright
```

Comment:
was this necessary? My understanding is that pyright can also be installed via pypi (uv). This will install the latest version of `pyright` from npm, which may be different from what's specified in `pyproject.toml`. 

Another thing is that, even if we install pyright from pypi, the package requires the system to have `node` installed. Ubuntu on GitHub actions comes with node, but since we're using `ubuntu-latest`, GitHub may switch to later versions of Node. I would use [actions/setup-node](https://github.com/actions/setup-node) to install node with version fixed to the current latest stable release (22.x)

Reply:
Thanks for the review! I misunderstood and thought I also needed to run `npm install pyright`. I'll add the `Set up Node.js` step.

Reply:
This is not resolved and is not necessary
https://github.com/emptymap/waddle/pull/62

Reply:
Sorry, I added node install and accidentally forgot to remove the original